doc/src/manual/methods.md

```diff
@@ -797,10 +797,38 @@ would be called only when the number of `indices` matches the dimensionality of
 When only the type of supplied arguments needs to be constrained `Vararg{T}` can be equivalently
 written as `T...`. For instance `f(x::Int...) = x` is a shorthand for `f(x::Vararg{Int}) = x`.
 
-## Note on Optional and keyword Arguments
+## Note on Optional/Default Positional and Keyword Arguments
 
-As mentioned briefly in [Functions](@ref man-functions), optional arguments are implemented as syntax for multiple
-method definitions. For example, this definition:
+There are two styles for setting optional agruments (a.k.a default values).  
+
+1. Using position arguments.
+1. Using keyword arguments. 
+
+Keyword arguments behave differently from ordinary positional arguments. 
+Specifically, they do not participate in method dispatch. 
+
+!!! note Function methods are dispatched based *only* on positional arguments. 
+Keyword arguments are processed after the matching method is identified.
+
+!!! warning A consequence of the above note on dispatch: 
```

Comment:
@StefanKarpinski: Would removing the `!!! warning` highlight be enough, or that whole paragraph?

Reply:
A note and a warning seems excessive. Maybe just write all of it inline? Also, it would make sense to reference https://github.com/JuliaLang/julia/issues/9498 as it’s known that there’s a bug here, but it would likely be too breaking to change it in a 1.x release.

Reply:
Hmm, delicate thing referencing buggy behavior in stable docs.  Is there a precedent you know of, off hand, that I can look at for style?